app/src/main/java/org/greenstand/android/TreeTracker/view/TextButton.kt

```diff
@@ -274,7 +274,7 @@ fun BoxScope.LanguageButton() {
         colors = AppButtonColors.ProgressGreen,
         modifier = Modifier
             .align(Alignment.Center)
-            .size(width = 100.dp, 60.dp),
+            .size(width = 120.dp, 60.dp),
```

Comment:
There may be other languages, so we need to find a way to handle them without increasing the size of the button.